vhost-device-vsock/src/vhu_vsock_thread.rs

```diff
@@ -437,26 +438,15 @@ impl VhostUserVsockThread {
 
     /// Read `CONNECT PORT_NUM\n` from the connected stream.
     fn read_local_stream_port(stream: &mut UnixStream) -> Result<u32> {
-        let mut buf = [0u8; 32];
-
-        // Minimum number of bytes we should be able to read
-        // Corresponds to 'CONNECT 0\n'
-        const MIN_READ_LEN: usize = 10;
+        const MAX_BUF: u64 = 32;
+        let mut buf = Vec::with_capacity(MAX_BUF as usize);
+        let mut reader = BufReader::new(stream).take(MAX_BUF);
```

Comment:
Why do we need `take()`?
Can't we just do that?
```rust
        let mut buf = Vec::new();
        let mut reader = BufReader::new(stream);
```

Reply:
even earlier the max buffer size was set to 32, i thought that was by design. `.tale()` prevents BufReader to error out if it goes beyond that. Small thing can basically be ignored also i guess

Reply:
It used to be needed because it was an unbuffered read (i.e. we were buffering), but now IMHO we can simply avoid that arbitrary limit and simplify the code.